Update apache-spark-load-data-run-query.md with PySpark3 kernel in Jupyter (#87706)
Azure/azure-content: articles/hdinsight/spark/apache-spark-load-data-run-query.md

1. Edit the URL `https://SPARKCLUSTER.azurehdinsight.net/jupyter` by replacing `SPARKCLUSTER` with the name of your Spark cluster. Then enter the edited URL in a web browser. If prompted, enter the cluster login credentials for the cluster.

> 2. From the Jupyter web page, **For the Spark 2.4** clusters,  Select **New** > **PySpark** to create a notebook. **For the Spark 3.1** release, Please Select **New** > **PySpark3** to create a notebook instead as PySpark kernel is no longer available in Spark 3.1. 

Reviewer: ```suggestion
2. From the Jupyter web page, **For the Spark 2.4** clusters,  Select **New** > **PySpark** to create a notebook. **For the Spark 3.1** release, select **New** > **PySpark3** instead to create a notebook because the PySpark kernel is no longer available in Spark 3.1. 
```
